rust/datafusion/src/optimizer/projection_push_down.rs

```diff
@@ -45,116 +48,12 @@ impl ProjectionPushDown {
     pub fn new() -> Self {
         Self {}
     }
-
-    fn optimize_plan(
```

Comment:
This just moves the function to outside `ProjectionPushDown`, since it does not really depend on the state of `ProjectionPushDown`, i.e. it can be written as a normal function.